sdk-setup/src/resize-rootfs

```diff
@@ -1,47 +1,65 @@
-#!/bin/bash
+#!/bin/sh
+#
+# Resize root FS to fully use the available space on the device
+#
+# Copyright (C) 2019 Jolla Ltd.
+# Contact: Martin Kampas <martin.kampas@jollamobile.com>
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# - Redistributions of source code must retain the above copyright
+#   notice, this list of conditions and the following disclaimer.
+# - Redistributions in binary form must reproduce the above copyright
+#   notice, this list of conditions and the following disclaimer in
+#   the documentation and/or other materials provided with the
+#   distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
 
-if [[ -e /resize_root_fs ]]; then
-   resize2fs /dev/sda1
-   rm /resize_root_fs	
-   exit
-fi
+set -o errexit
 
-pre=$(mktemp)
-post=$(mktemp)
+pipefail()
+{
+    case "${PIPESTATUS[*]}" in
```

Comment:
Unfortunately also PIPESTATUS is bash only :(
There are a few ugly solutions, e.g. here: https://stackoverflow.com/questions/8833396/pipe-command-output-but-keep-the-error-code

Reply:
That drives me crazy... :) thanks again